backend/utils/error_handler.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import status
+from typing import Dict, Tuple
+
+from models.api import APIError
+from settings import Constants
+
+ERROR_CODES_MAPPING: Dict[int, str] = {
+    20001: "Invalid request",
+    20002: "Unauthorized request",
+    20003: "Forbidden request",
+    20004: "Not found",
+    20005: "Internal server error",
+}
```

Comment:
You should change the structure like this:

```
ERROR_CODES: Dict[str, int] = {
    "INVALID_REQUEST": 20001
}
```

I don't remember the syntax in Python, you should check this again. And also, import this to a constant file, use this for returning error too

```
APIError(err_code=ERROR_CODE.INVALID_REQUEST)
```

Easier to read and maintain

Reply:
However, I think an ENUM is better for this scenario, you should check out this: https://www.geeksforgeeks.org/enum-in-python/

Reply:
Discuss later #15